kyaml/filesys/util_test.go

```diff
@@ -1,11 +1,14 @@
-package filesys_test
+// Copyright 2021 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+// +build !windows
+
+package filesys
```

Comment:
Why the change to not have a separate test package here and in a few other files?

Reply:
I had the same question but (maybe incorrectly) assumed that it didn't matter. Was there a motivation for having them separate?

Reply:
As far as I'm aware, the main reason for separate test packages is to force the tests to be written against the public API, i.e. test from the perspective of the consumers of the package. I can't speak to whether that was the reason for the separate package in this case. Since this PR is just moving code from one place to another, I think it's fair to ask for an explanation of for any actual changes that were included. :)